frigate/ffmpeg_presets.py

```diff
@@ -115,8 +117,12 @@ def get_selected_gpu(self) -> str:
     "preset-vaapi": "-r {0} -vf fps={0},scale_vaapi=w={1}:h={2},hwdownload,format=yuv420p",
     "preset-intel-qsv-h264": "-r {0} -vf vpp_qsv=framerate={0}:w={1}:h={2}:format=nv12,hwdownload,format=nv12,format=yuv420p",
     "preset-intel-qsv-h265": "-r {0} -vf vpp_qsv=framerate={0}:w={1}:h={2}:format=nv12,hwdownload,format=nv12,format=yuv420p",
+    # If `detect` is the only role using the decoded video, the four below are further optimized by adding
```

Comment:
Can you elaborate on this? Currently in frigate the only role that ever uses a decoded stream is detect. So if there is a performance benefit to using this resize arg instead of -vf scale_cuda then I think that should just be the default for the preset and no need for special casing

Reply:
You're right - the only corner case I can think of is if the user overrode the ffmpeg args to re-encode the video, but if they're doing that, then they're not using these presets anyway.

I had noticed that the ffmpeg cuvid decoder has a `-resize` option, so I added the same to the nvmpi decoder, allowing it to resize for free instead of doing an expensive upload/resize/download. I should add an `-fps` option too.

I now realize that frigate's nvidia preset doesn't even use the cuvid decoder, it uses the cuda accel, which does not have those flags - so I should move that comment to refer only to the two jetson presets and not the nvidia presets.

Anyway, concern addressed in latest commit.